website/src/components/Layout.tsx

```diff
@@ -52,7 +55,12 @@ export default class Layout extends React.PureComponent<LayoutProps> {
                     <meta property="og:image" content={metaProps.image} />
                     <meta property="og:description" content={metaProps.description} />
```

Comment:
Should this also use `seoDescription` considering that some other link unfurlers are likely to look at `<meta name="description">` first? Same for `twitter:*` tags and `*:title` tags
```suggestion
                    <meta property="og:description" content={metaProps.seoDescription || metaProps.description} />
```

Reply:
It wasn't clear from the Slack and sync conversations about this whether the `og:*` and `twitter:*` tags should reflect the search specific title/description, or the content on our site, so I was being conservative here. We can merge this for now, and I can follow up with stakeholders for more clarification.